okhttp-backend/monix/src/test/scala/sttp/client3/okhttp/monix/OkHttpMonixHttpTest.scala

```diff
@@ -17,6 +17,7 @@ class OkHttpMonixHttpTest extends HttpTest[Task] {
   override implicit val convertToFuture: ConvertToFuture[Task] = convertMonixTaskToFuture
 
   override def supportsDeflateWrapperChecking = false
+  override def supportsEmptyContentEncoding = false
```

Comment:
`false`, but there are some code changes in the backend?

Reply:
Yeah, it should not be changed to false. The test for that case was passing but I've got baited with an `OkHttp Dispatcher`  error and thought it might be something regarding the change. I've tested it locally and everything works fine.